src/Bookworm/Bookworm.php

```diff
@@ -63,9 +64,10 @@ public static function estimate($text, $type = self::TYPE_SHORT)
         $minutes = $minutes > 1 ? $minutes : 1;
 
         // Get the proper textual representation
-        $textual = self::$translations[$type == self::TYPE_SHORT ? 'min' : 'minute'];
-
-        return $minutes . ' ' . $textual;
+        if($type !== self::TYPE_NONE){
```

Comment:
Can you apply spaces as defined in [PSR-2](http://www.php-fig.org/psr/psr-2/) here?